lib/src/signed_video_h26x_auth.c

```diff
@@ -70,6 +70,22 @@ static const char *kAuthResultValidStr[SV_AUTH_NUM_SIGNED_GOP_VALID_STATES] = {"
     "SIGNATURE PRESENT", "NOT OK", "OK WITH MISSING INFO", "OK", "VERSION MISMATCH"};
 #endif
 
+/*
+ * Checks if linked hash validation is feasible at the start of validation.
+ * During the first validation slots, the hashes in |gop_info->linked_hashes| are all zeros,
+ * indicating that validation is not possible.
+ */
+static bool
+linked_hash_validation_is_not_feasible(signed_video_t *self)
```

Comment:
This function is called where you have access to "linked_hashes" and "hash_size", so it might be preferred to changed input parameters to "const uint8_t *linked_hashes, size_t hash_size"